magento/pwa-studio
# PWA Studio 12.1.0 release notes

## Description

12.1.0 Release notes for PWA Studio

## Related Issue

NA

Closes https://jira.corp.magento.com/browse/PWA-2335.

### Verification Stakeholders

- @supernova-at
- @jcalcaben 
- @davemacaulay  

### Specification




## Verification Steps

- Review wording of included features, bugs, ticket links, GitHub links, etc.
- Check breaking changes section to ensure I have captured all of them.

## Breaking Changes (if any)
